<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sprite Hero</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addFrameButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="animationPanelLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>821</width>
      <height>128</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="duplicateFrameButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicate Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearFrameButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Frame</string>
    </property>
   </widget>
   <widget class="QSlider" name="fpsSlider">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>12</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="fpsLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Phosphate</family>
     </font>
    </property>
    <property name="text">
     <string>FPS: 12</string>
    </property>
   </widget>
   <widget class="ColorChooser" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>481</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>70</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save SSP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open SSP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gifButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Export to GIF</string>
    </property>
   </widget>
   <widget class="SpriteView" name="widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorChooser</class>
   <extends>QWidget</extends>
   <header>colorchooser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpriteView</class>
   <extends>QLabel</extends>
   <header>spriteview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
